src/NearOrg/Notifications/Notification.jsx

```diff
@@ -259,10 +262,46 @@ const iconType = {
   "devgovgigs/reply": <i className="ph ph-share-fat" />,
 };
 
+const [isModalOpen, setIsModalOpen] = useState(false);
+
+const buildMenu = () => {
+  return [
+    {
+      name: (
+        <>
+          <i
+            className="ph-bold ph-bell-simple-slash"
+            style={{ color: "#D95C4A" }}
+          />
+          <span style={{ color: "#D95C4A" }}>Block</span>
```

Comment:
This should be visible only for users that enabled push notifications. It seems that checking whether a subscription endpoint exists should be enough to determine whether the push notifications have been enabled.

Reply:
I added a condition only for users that have enabled push notifications. Should be fixed in the latest commit. Thank you Milena.